frontend/interfaces/schedulable_query.ts

```diff
@@ -0,0 +1,111 @@
+import { IFormField } from "./form_field";
+import { IPack } from "./pack";
+import { IPlatformString } from "./platform";
+
+// Query itself
+export interface ISchedulableQuery {
```

Comment:
My personal preference would be to have the API request/response and react-query query key interfaces live next to the APIs in the `/services` files. And the general entity interfaces live in the `/interfaces` files and page-level interfaces like page props live alongside the page.

Reply:
@ghernandez345 @gillespi314 I did bring this up earlier in the sprint when proposing this interfaces doc, and Sarah and Rachel seemed okay with this at the time, though Gabe I think you may have missed that ping. It makes sense to me either way, but maybe for future iterations since all of our work this sprint is now based on these interfaces as they currently are.